sdk/node/test/transactionFeeds.js

```diff
@@ -0,0 +1,144 @@
+/* eslint-env mocha */
+
+const chain = require('../dist/index.js')
+const uuid = require('uuid')
+const assert = require('assert')
+const chai = require('chai')
+const chaiAsPromised = require('chai-as-promised')
+
+chai.use(chaiAsPromised)
+const expect = chai.expect
+
+const {
+  client,
+  createAccount,
+  createAsset,
+  signer
+} = require('../testHelpers/util')
+
+const buildSignSubmit = buildFunc =>
+  client.transactions.build(buildFunc)
+    .then(tpl => signer.sign(tpl))
+    .then(tpl => client.transactions.submit(tpl))
+
+describe('Transaction feeds', () => {
```

Comment:
Will do, although I notice that the file name itself is plural, so we should pick one noun style for everything and stick with it.